### DISCUSSION TITLE
Cannot specify fixed flows for all branches of internal ductnode

### DISCUSSION THREAD

**Discussion by Prometeo-ianic** [firemodels/fds#13418] [General]
Hi all,
I have these two fan, with 1 jet fan the simulation runs, without  
I get this error
ERROR(553): Cannot specify fixed flows for all branches of internal ductnode:Center Node 02 .

&DEVC ID='TIMER->OUT', QUANTITY='TIME', XYZ=0.0,0.0,0.0, SETPOINT=2.0/

&VENT ID='Inlet02', SURF_ID='HVAC', XB=38.2,38.6,46.3,46.3,2.4,2.8/ 
&VENT ID='Outlet02', SURF_ID='HVAC', XB=38.2,38.6,48.3,48.3,2.4,2.8, IOR=2/ 
&VENT ID='Inlet01', SURF_ID='HVAC', XB=27.6,28.0,45.8,45.8,2.4,2.8/ 
&VENT ID='Outlet01', SURF_ID='HVAC', XB=27.6,28.0,47.8,47.8,2.4,2.8/ 

&HVAC ID='Inlet Node 01', TYPE_ID='NODE', DUCT_ID='Duct_01_01', VENT_ID='Inlet01'/
&HVAC ID='Center Node 01', TYPE_ID='NODE', DUCT_ID='Duct_01_01','Duct_02_01', XYZ=27.8,46.8,2.6/
&HVAC ID='Outlet Node 01', TYPE_ID='NODE', DUCT_ID='Duct_02_01', VENT_ID='Outlet01'/
&HVAC ID='Duct_01_01', TYPE_ID='DUCT', DIAMETER=0.355, NODE_ID='Inlet Node 01','Center Node 01', ROUGHNESS=1.0E-3, LENGTH=0.990131, DEVC_ID='TIMER->OUT'/
&HVAC ID='Duct_02_01', TYPE_ID='DUCT', DIAMETER=0.355, FAN_ID='FAN_01', NODE_ID='Center Node 01','Outlet Node 01', ROUGHNESS=1.0E-3, LENGTH=1.009869, DEVC_ID='TIMER->OUT'/
&HVAC ID='Inlet Node 02', TYPE_ID='NODE', DUCT_ID='Duct_01_02', VENT_ID='Inlet02'/
&HVAC ID='Center Node 02', TYPE_ID='NODE', DUCT_ID='Duct_01_02','Duct_02_02', XYZ=38.4,47.3,2.6/
&HVAC ID='Outlet Node 02', TYPE_ID='NODE', DUCT_ID='Duct_02_02', VENT_ID='Outlet02'/
&HVAC ID='Duct_01_02', TYPE_ID='DUCT', DIAMETER=0.355, NODE_ID='Inlet Node 02','Center Node 02', ROUGHNESS=1.0E-3, DEVC_ID='TIMER->OUT'/
&HVAC ID='Duct_02_02', TYPE_ID='DUCT', DIAMETER=0.355, FAN_ID='FAN_02', NODE_ID='Center Node 02','Outlet Node 02', ROUGHNESS=1.0E-3, DEVC_ID='TIMER->OUT'/
&HVAC ID='FAN_01', TYPE_ID='FAN', TAU_FAN=-60.0, VOLUME_FLOW=3.136/
&HVAC ID='FAN_02', TYPE_ID='FAN', TAU_FAN=-60.0, VOLUME_FLOW=3.136/

What I am doing wrong? At the beginning I thought it was the fact that I should have used a different &HVAC ID for each fun, but it didn't work.
Thank you in advance

**Comment by mcgratta** [comment]
```
ERROR(553): Cannot specify fixed flows for all branches of internal ductnode:Center Node 02 .
```
What does the error code say?

**Reply by Prometeo-ianic** [reply]
I presume there is no alternative to prescribe a flow rate/pressure curve, isn't it? Sorry, but some years ago it was possible to set fixed flows for all branches. Why is it changed? (just curiosity)

**Reply by drjfloyd** [reply]
You cannot prescribe all flow in and out of a duct node. This is done to ensure mass conversaton at the limit of machine precision since there is some error in converting reals to binary.

**Reply by drjfloyd** [reply]
I see now that you appear to have two jet fans each of which are an inlet node, a duct, an internal node, a duct with a fan, and an exit node. This should pass the check as the internal nodes have a fan on one side and nothing specified on the other.  I'll see if I can reproduce this with a simple input.

**Reply by Prometeo-ianic** [reply]
Thanks @drjfloyd , but I am not understanding which is the best strategies to improve it, avoiding the error.  A strategy that seems viable now can be use a common flow rate/pressure. Are there alternatives? If the problem is the mass conservation, why one jet fan gets causes an error and two jet fan cause the error? sorry in advance for the questions

**Reply by drjfloyd** [reply]
There might be a bug.

**Reply by drjfloyd** [reply]
I've tried making a simple case that has the same topology for the HVAC network but I don't get an error.  Can you attach your full input?

**Reply by Prometeo-ianic** [reply]
I did the same, and I start progressively deleting things until the problem disappear. I find that some obstruction over the jet fans that I used to avoid lateral dispersion of the flow rate gave this problem, that some years ago did not happen. Thank you for the help. I really appreciate

**Reply by drjfloyd** [reply]
Can you still provide the input file? If this was due to an obstruction blocking the VENT, then I'd want to see if we can throw a more meaningful error message.

**Reply by Prometeo-ianic** [reply]
[JetFan.txt](https://github.com/user-attachments/files/17009645/JetFan.txt)
Sorry, but the file was too large and I had to simplify it as much as I can

**Reply by drjfloyd** [reply]
The case you attached runs for me with FDS 6.9.1.  Are you running an older version?
